README.md

```diff
@@ -1,5 +1,13 @@
-# Fragments-Download
-This script downloads anyurl by fragmenting the file and increases speed drastically.
+# All purpose downloader 
+## Powered by CRUx - The computing and programming club 
 
-usage: python3 frag.py URL
```

Comment:
Add usage in its own section: Dependency installation etc? Also, use single tildes for writing code snippets like: `python3 ...`